scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -230,14 +230,14 @@ workflow case_gatk_acnv_workflow {
           mem=4
     }
 
-    call CNLoHAndSplitsCaller {
+    call AllelicBalanceCaller {
```

Comment:
The use of "split" vs. "balance" throughout this PR is somewhat confusing.  For example, the output directory and task name in the WDL are call_balanced_dir and AllelicBalanceCaller, respectively, but the CLI is named CallAllelicSplits.

I think it is clear what a "balanced call" should be (is this segment balanced or not?).  It's less clear what a "split" is supposed to represent from the name alone; from AllelicSplitCall.java, it seems to represent quite a lot of quantities.  Perhaps choose a more evocative name?

Note that this was probably a consequence of the fact that "CNLoHCall" was already trying to represent more than its name suggests.